tests/evaluation/empty_relations2/empty_relations2.err

```diff
@@ -0,0 +1,3 @@
+Warning: No rules/facts defined for relation empty in file empty.dl at line 14
```

Comment:
```suggestion
Warning: No rules/facts defined for relation empty in file empty_relations2.dl at line 14
```

Reply:
Hahahahahha this typo has plagued my existence, cheers for the catch